src/get-survey-results.js

```diff
@@ -33,7 +33,8 @@ async function waitForBuildToComplete(token, dataCenter, progressId) {
   if (data.result.percentComplete === 100) {
     return data.result.file;
   }
-
+  //wait for a second
+  Atomics.wait(new Int32Array(new SharedArrayBuffer(4)), 0, 0, 1000);
   return waitForBuildToComplete(token, dataCenter, progressId);
```

Comment:
Maybe this instead?:

```suggestion
  // Try again after one second.
  return await new Promise(() => { setTimeout(waitForBuildToComplete, 1000, token, dataCenter, progressId); });
```

 ^^^^^untested

Reply:
I'm just eyeballing it, but I don't think that promise would ever resolve. I don't know, I can never reason about `setTimeout` effectivly. I can switch it up to something like https://stackoverflow.com/a/41957152/796999 instead. I agree, I don't know anything about Atomics.wait. Total copy paste job.

Reply:
Whoops, you're right.

Let me check that I'm not completely wrong about Atomics....